src/EA.Weee.Xml/MemberRegistration/v3schema.xsd

```diff
@@ -35,6 +35,7 @@
 =   06/07/2015      Owen H               3.05       Removed unbounded maxOccurs constraint from overseasContact on overseasProducerType.
 =   15/07/2015      Owen H               3.06       Removed unused billingAddressType.
 =   07/10/2015      Graham A-T           3.07       Updated producerType to make authorisedRepresentative optional.
+=	14/03/2025		Sreedhar			 3.08		Amended sellingTechnique (Added Online Marketplaces and Fulfilment Houses)
```

Comment:
I also think it's a bit weird to have personal names in this -- I'm not sure if this is end-user facing or not!